modules/cnv-viewing-network-state-of-node.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-observing-node-network-state.adoc
+
+[id="cnv-viewing-network-state-of-node_{context}"]
+= Viewing the network state of a node
+
+A `NodeNetworkState` object exists on every node in the cluster. This object is periodically updated and captures the state of the network for that node. 
+
+.Procedure
+
+. List all the `NodeNetworkState` objects in the cluster:
++
+----
+$ oc get nns
+----
+
+. Inspect a `NodeNetworkState` to view the network on that node. The output in this example has been redacted for clarity:
++
+----
+$ oc get nns node01 -o yaml
+----
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkState
+metadata:
+  name: node01 <1>
+status:
+  currentState: <2>
+    dns-resolver:
+...
+    interfaces:
+...
+    route-rules:
+...
+    routes:
+...
+  lastSuccessfulUpdateTime: "2020-01-31T12:14:00Z" <3>
+----
+<1> Name of the `NodeNetworkState` is taken from the node.
+<2> `currentState` contains the DNS configuration, lists the interfaces and their configuration, and the routes.
```

Comment:
This sounds ambiguous to me.

Reply:
Changed to:
"The `currentState` contains the complete network configuration for the node, including the DNS, interfaces, and routes."